src/player.rs

```diff
@@ -24,4 +24,18 @@ impl Player {
             Black => White,
         }
     }
+
+    pub fn is_white(self) -> bool {
```

Comment:
I feel like these functions could probably be removed

Reply:
yep! 3482d30ae03516a0cbc0f749b986b94ed60e5db0